pkgs/development/python-modules/bbox/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyquaternion
+}:
+
+buildPythonPackage rec {
+  pname = "bbox";
+  version = "0.9.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-ucR7mg9eubEefjC7ratEgrb9h++a26z8KV38n3N2kcw=";
+  };
+
+  propagatedBuildInputs = [ pyquaternion ];
```

Comment:
`numpy` is also mentioned as [runtime dependency](https://github.com/varunagrawal/bbox/blob/master/pyproject.toml#L10).